kratos/includes/gid_mesh_container.h

```diff
@@ -183,8 +183,8 @@ class GidMeshContainer
                                 || mGeometryType == GeometryData::Kratos_Line3D3 )
                         {
                             nodes_id[0] = (it)->GetGeometry() [0].Id();
-                            nodes_id[1] = (it)->GetGeometry() [2].Id();
```

Comment:
are u sure of this change? i think that the numbering 0 2 1 is the correct one ... at least quadratic elements are done this way in GiD

Reply:
Yes, GiD suplies  (0, 2, 1) so, (Geometry[0]-> 0, Geometry[1] -> 2 ,Geometry[2] -> 1), then GiD expects (0, 2, 1), so (Geometry[0], Geometry[1], Geometry[2]).  I realised that the order was changed here because the output I got in GiD was wrong. If somebody is doing something internally with the local numbering, then he must say something.
Without the correction:
![incorrect](https://user-images.githubusercontent.com/23210788/37434860-641c0cbc-27e1-11e8-803e-e2ccece14b51.jpg)
With the correction:
![correct](https://user-images.githubusercontent.com/23210788/37434881-741c1ff8-27e1-11e8-8a62-3ea3ed884db0.jpg)